### PR TITLE
Add FAQ on how to activate format=flowed

### PR DESCRIPTION
**Pull request by cpbotha**
PR as requested in https://github.com/djcb/mu/issues/569

### CODE REVIEW

**Comment by djcb** [comment]
Merged -- thanks!

File: mu4e/mu4e.texi [hunk 1]
```diff
@@ -3389,6 +3389,34 @@ Sending...done
...
+indicate intra-paragraph breaks with soft newlines and inter-paragraph
+breaks with hard newlines. When the Gnus code sees these on outgoing
+emails, it will automatically set @t{format=flowed}.
+
+To enable this, you can use something like this in your init.el:
+
+@lisp
+;; tip submitted by mu4e user cpbotha
+(add-hook 'mu4e-compose-mode-hook
+          "Outgoing mails get format=flowed."
```
Comment (DamienCassou) [inline mu4e/mu4e.texi]: shouldn't this documentation be part of the function instead?

Comment (xmaillard) [inline mu4e/mu4e.texi]: @DamienCassou +1

**Comment by aspiers** [comment]
I've linked to this FAQ from https://www.emacswiki.org/emacs/FormatFlowed

**Pull request merged**